src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -437,7 +638,7 @@ protected Solution CreateOrOpenSolution(bool nullPaths = false)
             _persistentLocationService?.RaiseShutdown();
             _persistentLocationService = new MockPersistentStorageLocationService(solution.Id, _persistentFolder);
 
-            var storage = GetStorageService(_persistentLocationService, solutionSizeTrackerMock.Object, faultInjectorOpt).GetStorage(solution);
+            var storage = GetStorageService(_persistentLocationService, solutionSizeTrackerMock.Object, faultInjectorOpt).GetStorage(solution, checkBranchId: true);
```

Comment:
what made we now need the flag when we didn't need it before?

Reply:
i prefer the explicitness.  i'm not a fan of the original method which just hardcoded in this behavior.  But we can't change it because it's public.  So this way the consumers are explicit about the behavior they want.